src/test/regress/pg_regress_multi.pl

```diff
@@ -251,6 +251,18 @@ END
     close $fh;
 }
 
+sub write_settings_to_postgres_conf 
+{
+    my ($pgOptions, $pgConfigPath) = @_;
+    print $pgConfigPath;
+    open(my $fd, ">>", $pgConfigPath);
```

Comment:
do files need to be closed?

Reply:
>After processing the file such as reading or writing, you should always close it explicitly by using the close() function. If you don't, Perl will automatically close the file for you, however, it is not a good programming practice. The filehandle should always be closed explicitly.

I guess we should close it to be a good perl programmer...